### ISSUE TITLE
StepLineSeries does not 'step' when Angular AOT is enabled

### ISSUE THREAD

**Issue by samkristoff** [amcharts/amcharts4#2180]
Hi,

I'm using amCharts v4.9.1 in an angular 6 app and have a handful of charts working well.

However, I have a StepLineSeries on a chart that looks great in dev, but looks like is reverting to a regular LineSeries (maybe?) when built with AOT enabled.

This chart in particular has multiple YAxis, one for the 'analog data' on top and one for each 'digital' series on the bottom.  In the screen shot below this 2 axis total.

Here is a stripped down version of the code that creates the 'digital' axis and series.  I haven't found any settings that change this behavior:
```
let valueAxis = this.chart.yAxes.push(new am4charts.ValueAxis());
valueAxis.title.fontSize = this.labelFontSize;
valueAxis.tooltip.animationDuration = 0;
valueAxis.cursorTooltipEnabled = false;
valueAxis.renderer.minWidth = 35;
valueAxis.hide();
valueAxis.height = am4core.percent(4);

let series = this.chart.series.push(new am4charts.StepLineSeries());
series.yAxis = valueAxis;
series.strokeWidth = 3;
series.name = context;
series.dataFields.dateX = "epochTime";
series.dataFields.valueY = context;

series.connect = false;
series.autoGapCount = 4;
const labels = this.parseSecondaryIndexContext(context);
series.tooltipText = `${labels.short}`;
series.legendSettings.labelText = labels.long;
series.interpolationDuration = 0;
series.strokeOpacity = 0;
series.fillOpacity = 0.3;
```

With AOT disabled:
![image](https://user-images.githubusercontent.com/1994983/75381533-c1022e00-588d-11ea-8c22-cbf66bff1d22.png)

With AOT enabled:
![image](https://user-images.githubusercontent.com/1994983/75381616-e858fb00-588d-11ea-8aa4-2d84e3b2f0f3.png)

Possibly related to #1532 although changing connect and autoGapCount didn't have any effect.

Thanks!

**Comment by Pauan** [comment]
@samkristoff I haven't been able to reproduce this, could you please share more of your chart code, and the data that you're using? It's okay if it's fake data, I just need to be able to reproduce the problem.

**Comment by samkristoff** [comment]
Hey Pauan,

I created a new angular project and moved over as little as possible to reproduce the issue and the stepped line is working in the new project as expected even when buildings with AOT.  I'll pull in more of my original code to see if I can get it reproducing in the test project.

**Comment by samkristoff** [comment]
I found the issue.

I created an angular component that contains an amchart instance and I use this component in different places in my app.

In the component I included `am4core.options.minPolylineStep = 1;`
However the page containing the component also included `am4core.options.minPolylineStep = 20;`  (a hold over from some previous testing).

The setting in the page seems to override the setting in the component only when AOT is enabled.

I removed the setting from the page and everything works as desired.
